ocs_ci/utility/kms.py

```diff
@@ -1264,6 +1265,101 @@ def create_hpcs_csi_kms_connection_details(
         csi_kms_conn_details["metadata"]["namespace"] = namespace
         self.create_resource(csi_kms_conn_details, prefix="csikmsconn")
 
+    def cleanup(self):
+        """
+        Cleanup the backend resources in case of external
+
+        """
+        # nothing to cleanup as of now
+        logger.warning("Nothing to cleanup from HPCS")
```

Comment:
is it dummy function or we miss to cleanup ?

Reply:
Hi @vavuthu , this is a dummy function and has nothing specific to cleanup in case of HPCS as of now.